smurfs/src/components/Jumbotron.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Button, Jumbotron } from "reactstrap";
+
+const Jumbo = () => {
+  return (
+    <Jumbotron>
+      <h1 className="display-3">Hello, world!</h1>
+      <p className="lead">This is a simple CRUD app using Async Redux</p>
+      <hr className="my-2" />
+      <p>
+        It uses Redux Thunk to dispatch asyncronous AXIOS calls from actions
+      </p>
+      <p className="lead">
+        <a href="mailto:mcferrenmax@gmai.com">
```

Comment:
People are going to be really disappointed when they try to hire you and your email doesn't actually go anywhere :(

Reply:
doh!